relayer/chains/sui/keys.go

```diff
@@ -109,5 +109,5 @@ func (p *Provider) ImportKeystore(ctx context.Context, keyPath, passphrase strin
 
 // keystorePath is the path to the keystore file
 func (p *Provider) keystorePath(addr string) string {
-	return path.Join(p.cfg.HomeDir, "keystore", p.NID(), addr)
```

Comment:
since we already have NID folder for general wallet key, should we wrap it inside wallet folder as this would mean breaking changes which doesn't seem required.

Reply:
I was planning to write a migration script to facilitate the breaking change.

The keystore location seems logical because it houses all the keys inside.

Reply:
yes but is the new "wallet" folder really required ? All the NID folders contains wallet and the new one with cluster identifier contains cluster key.

Reply:
Not really, the reason behind using the "wallet" dir structure is to segregate the keys, some sort of house keeping I would say. 

We need to walk through inside the keystore dir for listing and there's no way correct way to identify whether it is wallet, except using hack.

Reply:
/keystore/{nid}/{wallet-adddress} for chains keystore is already fine. For relayer keystore we can just add another sub-folder inside keystore folder like: /keystore/relayer/{pub-key}

Reply:
> Not really, the reason behind using the "wallet" dir structure is to segregate the keys, some sort of house keeping I would say.
> 
> We need to walk through inside the keystore dir for listing and there's no way correct way to identify whether it is wallet, except using hack.

We can list by directly accessing the filePath because we can identify it deterministically for all chains. We don't need to walk through the sub folders.

Reply:
We can but currently we don't.